modules/serverless-logic-cluster-scoped-eventing-system-config.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-workflow-eventing-system
+
+:_mod-docs-content-type: CONCEPT
+[id="serverless-logic-cluster-scoped-eventing-system-config_{context}"]
+= Cluster-scoped eventing system configuration
+
+In a `SonataFlowClusterPlatform` setup, workflows are automatically linked to the Broker specified in the associated `SonataFlowPlatform` CR. This linkage follows the Eventing System Configuration Precedence Rules.
+
+To ensure proper integration, you can configure the Broker in the `SonataFlowPlatform` CR referenced by the `SonataFlowClusterPlatform` CR.
+
+The following example displays how to configure the `SonataFlowClusterPlatform` CR and its reference to the `SonataFlowPlatform` CR:
+
+.Example of cluster-scoped eventing system configuration 
```

Comment:
Given my previous comment, this heading might no longer be needed as a lead-in sentence should cover what the example is primarily showcasing.